pkgs/development/libraries/libspatialaudio/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:
```

Sorry about this, but this looks really strange. Like a giraffe without neck or something.

```suggestion
{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:

```

---

Hah, someone needs to add this to nixfmt :P